tinydrive/tinydrive-api/standalone.md

```diff
@@ -51,10 +74,38 @@ The URL for the selected file would be in the following format: `https://drive.t
 
 The modification date for the file in ISO 8601 format for example `2019-02-24T15:00:00Z`
 
+## `tinydrive.controls.pick`
+
+The `tinydrive.controls.pick` method enables you to pick files from {{site.cloudfilemanager}} and get the meta data of those files returned in a callback. For a complete list of available settings to pass into this method check the [Picker settings](#pickersettings) section in this page. A control object is returned which control the {{site.cloudfilemanager}} lifecycle, which it no longer does by itself.
+
+### File Picker Result Format
```

Comment:
```suggestion
### File picker result format
```
We use sentence case for titles normally. I realise this is incorrect in the version you copied from as well, unfortunately, I don't believe these docs have been properly reviewed/edited.